scripts/format-all.sh

```
@@ -0,0 +1,40 @@
+        davidanson/markdownlint-cli2:latest \
+        --config /workspace/.markdownlint.json \
+        --fix \
+        "docs/**/*.md" "*.md" 2>/dev/null || echo "  Some markdown files had formatting issues (non-critical)"
+else
+    echo "⚠️  Docker not found. Skipping Markdown formatting."
+fi
+
+# Fix EditorConfig issues
+echo "⚙️  Fixing EditorConfig issues..."
+# Remove trailing whitespace
+find . -name "*.kt" -o -name "*.md" -o -name "*.yml" -o -name "*.yaml" -o -name "*.json" | \
```

Shellcheck (0.10.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)